src/modelmanager.cpp

```diff
@@ -749,10 +750,35 @@ Status ModelManager::reloadModelVersions(std::shared_ptr<ovms::Model>& model, st
 
 Status ModelManager::reloadModelWithVersions(ModelConfig& config) {
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Started applying config changes to model: {}", config.getName());
+
+    if (config.isStateful() && config.isDynamicParameterEnabled()) {
+        SPDLOG_LOGGER_ERROR(modelmanager_logger, "Requested setting dynamic parameters for stateful model {}. Dynamic shape and dynamic batch size not supported for stateful models.", config.getName());
+        return StatusCode::REQUESTED_DYNAMIC_PARAMETERS_ON_STATEFUL_MODEL;
+    }
+    if (!config.isStateful()) {
+        if (config.getMaxSequenceNumber() != DEFAULT_MAX_SEQUENCE_NUMBER) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Requested setting max sequence number parameter for non stateful model {}.", config.getName());
+            return StatusCode::INVALID_NON_STATEFUL_MODEL_PARAMETER;
+        }
+        if (config.getSequenceTimeout() != DEFAULT_SEQUENCE_TIMEOUT_SECONDS) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Requested setting sequencetimeout parameter for non stateful model {}.", config.getName());
+            return StatusCode::INVALID_NON_STATEFUL_MODEL_PARAMETER;
+        }
+        if (config.isLowLatencyTransformationUsed() != false) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Requested low latency transformation parameter for non stateful model {}.", config.getName());
+            return StatusCode::INVALID_NON_STATEFUL_MODEL_PARAMETER;
+        }
+    }
```

Comment:
If its valdated in NodeParse i think validation here is not required.
Changing types is not a good idea. We could do this by adding additional config fields but I think it would make validation even more complex and the flow not clear. So we can remove those checks for uint values.